<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BatchPatternCapture</class>
 <widget class="QMainWindow" name="BatchPatternCapture">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Batch Pattern Capture</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="centralLayout">
    <item>
     <layout class="QHBoxLayout" name="fileLayout">
      <item>
       <widget class="QLabel" name="layoutFileLabel">
        <property name="text">
         <string>Layout File:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="layoutFileEdit"/>
      </item>
      <item>
       <widget class="QPushButton" name="browseButton">
        <property name="text">
         <string>Browse</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="maskLayerLabel">
      <property name="text">
       <string>Mask Layer Number:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="maskLayerEdit">
      <property name="text">
       <string>1</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="inputLayersLabel">
      <property name="text">
       <string>Input Layers Numbers:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="inputLayersEdit">
      <property name="text">
       <string>2</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="dbNameLabel">
      <property name="text">
       <string>Database Name:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="dbNameEdit">
      <property name="text">
       <string>test_db</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="runButton">
      <property name="text">
       <string>Run DFM Pattern Capture</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="cancelButton">
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="logText">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
